db/re/item_db_equip.yml

```
@@ -102861,7 +102728,7 @@ Body:
     EquipLevelMin: 99
     Refineable: true
     Script: |
-      bonus2 bAddClass,Class_All,2+(getrefine()/3);
+      bonus2 bAddClass,Class_All,3+(getrefine()/3);
```

It was fine according to KRO description
```suggestion
      bonus2 bAddClass,Class_All,2+(getrefine()/3);
```

---

Thanks